packages/orbit-tailwind-preset/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@kiwicom/orbit-tailwind-preset",
+  "version": "0.0.1",
+  "description": "Orbit tailwind preset which is based on orbit-design-tokens",
+  "author": "Kiwi.com",
+  "license": "MIT",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kiwicom/orbit",
+    "directory": "packages/orbit-tailwind-preset"
+  },
+  "files": [
+    "README.md",
+    "dist/*"
+  ],
+  "keywords": [
+    "orbit",
+    "tailwind",
+    "preset",
+    "design-system",
+    "tokens"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "pretest": "yarn workspace @kiwicom/orbit-design-tokens build && yarn build && yarn tailwindcss -o style.css",
+    "test": "jest"
+  },
+  "peerDependencies": {
+    "tailwindcss": ">=3.3.2"
+  },
+  "dependencies": {
+    "@kiwicom/orbit-design-tokens": "*"
```

Comment:
As agreed, this should actually start on the version of orbit-design-tokens that is compatible with tailwind

Reply:
that will be possible only after publishing it to npm, otherwise on CI it will be broken here